File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/ItemsCreateItemPhotoRequestItem.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class ItemsCreateItemPhotoRequestItem {
    @JsonProperty("EncodedImageBase64")
    public String encodedImageBase64;
    @JsonProperty("FileName")
    public String fileName;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/ItemsCreateItemPhotoRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class ItemsCreateItemPhotoRequestItem(
    @JsonProperty("encodedImageBase64")
    val encodedImageBase64: String? = null,
    @JsonProperty("fileName")
    val fileName: String? = null
)
